include/dxc/dxcapi.h

```diff
@@ -477,10 +477,17 @@ typedef enum DXC_OUT_KIND {
   DXC_OUT_ROOT_SIGNATURE = 9, // IDxcBlob - Serialized root signature output
   DXC_OUT_EXTRA_OUTPUTS  = 10,// IDxcExtraResults - Extra outputs
   DXC_OUT_REMARKS = 11,       // IDxcBlobUtf8 or IDxcBlobUtf16 - text directed at stdout
+  DXC_OUT_TIME_REPORT = 12,   // IDxcBlobUtf8 or IDxcBlobUtf16 - text directed at stdout
```

Comment:
The comment on this one and the prior one should say `IDxcBlobWide` instead of `IDxcBlobUtf16`.  I'm not sure how the prior one ended up with the older comment either.